cover.html.twig

```diff
@@ -1,5 +1,5 @@
 <section class="cover{% if size|default(false) %} cover--{{ size }}{% endif %}{% if overlay|default(false) %} cover--overlay{% endif %}{% if rounded|default(false) %} cover--rounded{% endif %} {{ class|default('') }}"{% if id is defined %} id="{{ id }}"{% endif %}{% if height is defined %} style="height: {{ height }};"{% endif %}>
-    <div class="cover__bg{% if parallax|default(false) %} cover__bg--parallax{% endif %}"{% if background is defined %} style="background-image: url('{{ background }}');"{% endif %}></div>
+    <div class="cover__bg{% if parallax|default(false) %} cover__bg--parallax{% endif %}"{% if background is defined %} style="background-image: {% if opacity|default(null) %}{{ opacity ~ ', ' }}{% endif %}url('{{ background }}');"{% endif %}></div>
```

Comment:
Is it not supposed to be :

```twig
style="{% if background|default(null) %}background-image: url('{{ background }}');{% endif %}{% if opacity|default(null) %}opacity: {{ opacity }};{% endif %}"
````

style property like `background-image: 0.5, url("paper.gif");` not supposed to work